src/patternfly/components/Page/page-main-wizard.hbs

```diff
@@ -0,0 +1,6 @@
+<section class="pf-c-page__main-wizard{{#if page-main-wizard--modifier}} {{page-main-wizard--modifier}}{{/if}}"
```

Comment:
Did you decide to go with using the page main section here or this new element?

Reply:
New element. To make it work with the main page section would require setting a `flex-basis` or `height`, which could be a breaking change.